client/landing/stepper/declarative-flow/hosted-site-migration-v2/index.tsx

```diff
@@ -86,6 +120,83 @@ export default {
 						)
 					);
 				}
+
+				if ( currentStep === SITE_MIGRATION_UPGRADE_PLAN.slug ) {
```

Comment:
This massive amount of money is not good, but for now, at least, the automated tests will ensure an easy refactor on the next ticket.

Reply:
In the same refactor I also tried to improve it a bit, extracting it to an external hook, which helped with the indentation, and also organizing it in an object.

https://github.com/Automattic/wp-calypso/pull/93382/commits/05dc80ba99bc2c43c717c5d77f25953c80120dd6